app/services/import_polygons.rb

```diff
@@ -41,4 +43,57 @@ def location_categories_include?(region_name)
       location_type == :london_boroughs && DOWNCASE_BOROUGHS.include?(region_name) ||
       location_type == :cities && DOWNCASE_CITIES.include?(region_name)
   end
+
+  def get_buffers(points)
+    # ArcGIS API documentation: https://ons-inspire.esriuk.com/arcgis/sdk/rest/index.html#//02ss0000003z000000
+    buffer_distances_in_miles = [5, 10, 15, 20, 25]
+    buffered_boundaries = {}
+    buffer_distances_in_miles.each do |distance|
+      # convert 1D array, to 2D array for arcgis
+      polygon_coords = points.each_slice(2).to_a
+      response = HTTParty.get(buffer_api_endpoint(polygon_coords, convert_miles_to_metres(distance)))
+      buffer_coords = response.dig("geometries", 0, "rings").flatten
+      # algolia requires a 2D containing 1 element with all the coordinates contained within it
+      # so buffer_coords array has been wrapped in another array
+      buffered_boundaries[distance.to_s] = format_buffer_coords_for_algolia(buffer_coords)
```

Comment:
You don't need to do this. We store the 1d array and in `app/services/search/location_polygon.rb:41` we put it in the format algolia requires